judge/bridge/judgelist.py

```diff
@@ -47,6 +47,12 @@ def register(self, judge):
             self.judges.add(judge)
             self._handle_free_judge(judge)
 
+    def disconnect(self, judge_id, force=False):
+        for judge in self.judges:
+            if judge.name == judge_id:
+                judge.disconnect(force=force)
+                return
```

Comment:
Sometimes, a judge might end up connecting multiple times. I think killing that brain damage would be nice to have, so maybe we shouldn't `return`.